.vscode/AccountManager.java

```diff
@@ -0,0 +1,142 @@
+import java.util.HashMap;
+import java.util.Scanner;
+
```

Comment:
1. Using Constants for Repeated Values:

"The first suggestion I have is regarding the repeated string values, such as menu options in your AccountManager class. Instead of writing these values multiple times, you could store them in constants. This makes it easier to update them in the future and ensures that you don't make typos when using them.
-
2. Improving Error Handling:

"Another thing I noticed is that the error handling could be a bit more informative. For example, when a user enters the wrong password or tries to delete a non-existing account, you could provide more specific messages to guide the user.

-
3. Using More Modular Methods:

"My last suggestion is to make your code even more modular by breaking some larger methods into smaller ones. For example, your displayInitialMenu() method does a lot of things: it displays the menu, takes user input, and handles the logic for registration or login. It could be cleaner and easier to maintain if these tasks were split into separate methods